### DISCUSSION TITLE
How can the community help?

### DISCUSSION THREAD

**Discussion by YvanDaSilva** [sqlc-dev/sqlc#3490] [Issue Triage] [answered]
I see sqlc repo getting quite a bit of stale issues and PR's.

How can the community contribute to help you @kyleconroy.
Who is currently allowed to review, merge PR's, etc.
Is there a place where you share sqlc long term vision? The discord also seems dead.

It would be a shame to see so much work go to waste.
If you need help, don't hesitate to reach out.

If there is a better place for this discussion, let me/us know.

**Comment by kyleconroy** [comment, marked as answer]
@YvanDaSilva you're right that development has slowed a bit, but I'm still here slowly triaging things. We still have a release planned for later this month, although it's a bit later than I had hoped.

sqlc has been an open source project for almost four years. Don't fret, it's not going anywhere.

**Reply by YvanDaSilva** [reply]
@kyleconroy thank you very much for your work and answer.

And apologies for my slow reply, I failed to see a notification for this reply...

Super glad to hear that.

Feel free to reach out if you need some help.
On my end I would gladly work on adding support for sqlc.restruct macro or anything alike that would allow sqlc users a better control of how generators should serialize data structures from db rows.
i.e. add support for []slices mapping.

Keep up the great work.
Cheers.
